cogs/music.py

```diff
@@ -970,23 +970,19 @@ async def get_song_info(self) -> str:
 		return f"{self.song.title} - {self.song.url}"
 
 
-def validate_url(url):
+async def validate_url(url):
 	"""
 	Checks to see if url has any valid extractors for youtube_dl
 
 	Parameters:
 		- url: the url to search for extractors.
 
 	Returns:
-		-True, if site has dedicated extractor
-		-False, if site has no dedicated extractor
+		-extractor, if site has dedicated extractor
```

Comment:
It returns a bool (True/False) rather than an extractor.

Reply:
Are you sure? I thought it returns an object, but it evaluates to true.

Reply:
I might have misread tho.

Reply:
Actually nevermind I think your right.